tests/integrations/docker/test_verify_docker.py

```diff
@@ -54,6 +59,7 @@ def test_docker_doesnt_exist(test_command):
     )
 
 
+@mock.patch('briefcase.integrations.docker._verify_docker_can_run')
```

Comment:
Preference is to avoid using mock.patch, and validate *all* the calls made to services.

Reply:
Here I'm not sure which would be preferred.  I could:

1) Remove the patch.  But since both the call to `docker --version` and the call to `docker info` will then get the return code 1 output, this test will no longer return the docker wrapper, `_verify_docker_can_run` will see a failure and raise an exception instead.  This may be desired, since I don't know if there is a reasonable set of circumstances in which `docker --version` fails and `docker info` succeeds

2) Make the side effect of this a more complicated function that switches the return code based on the arguments, so that `--version` can fail and `info` can succeed.  This would preserve the spirit of the original test.

Initially it sounds like you'd prefer 2; if so, can do.  Just wanted to get your thoughts on whether this is still a valid test circumstance